framework/src/controls/Control.C

```diff
@@ -23,10 +23,8 @@ validParams<Control>()
   params += validParams<TransientInterface>();
   params += validParams<SetupInterface>();
   params += validParams<FunctionInterface>();
+  MooseUtils::setExecuteOnFlags(params, {EXEC_INITIAL, EXEC_TIMESTEP_END});
```

Comment:
If these flags are used as flags, these kinds of operations become EXEC_INITIAL | EXEC_TIMESTEP_END, no more `std::set` or `std::vector`